znai-reactjs/src/doc-elements/search/SearchBox.jsx

```diff
@@ -1,35 +1,56 @@
-const React = require('react');
+/*
```

Comment:
since we are touching this file for the first time since two sigma, please add znai maintainers 2025 to the copyright (see other files as an example)